ui/src/callWithTelemetryAndErrorHandling.ts

```diff
@@ -94,8 +94,13 @@ function handleError(context: IActionContext, callbackId: string, error: unknown
             message = errorData.message;
         }
 
+        const items: MessageItem[] =
```

Comment:
Could you construct just an empty array and then push in the buttons as we need them?  I was thinking for the future, it'd be easier to push extra buttons based on errorHandling conditions if we do it that way.